### ISSUE TITLE
How to use?

### ISSUE THREAD

**Issue by ItzLoonie** [MrSypz/Crital#12]
Mod page doesn't explain how to use the mod at all.
In game there's this Grinder Table block that has three slots.
One holds Copper, no idea what goes in the other slots.

I assumed equipment but no, those don't go in there.

This mod is way too confusing.

**Comment by MrSypz** [comment]
You're right, This mod is currently in beta, with many features still changing. I'll provide a detailed description once it's released.
